cmd/kube-apiserver/app/server.go

```diff
@@ -560,6 +561,7 @@ func BuildStorageFactory(s *options.ServerRunOptions) (*serverstorage.DefaultSto
 		[]schema.GroupVersionResource{
 			batch.Resource("cronjobs").WithVersion("v1beta1"),
 			storage.Resource("volumeattachments").WithVersion("v1alpha1"),
+			admissionregistration.Resource("initializerconfigurations").WithVersion("v1alpha1"),
```

Comment:
What exactly are the things in this list?

Reply:
These are the objects that need to be serialized to a different version before storing to etcd than the other objects in the same group.

Reply:
Since it looks like there's still a test to fix, can you add a comment to that effect?

Reply:
Done.